.circleci/config.yml

```diff
@@ -0,0 +1,66 @@
+
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+  prepare:
+    docker:
+      - image: mbedos/mbed-os-env:latest
+    working_directory: ~
+    steps:
+      - checkout:
+          path: mbed-os-example-cellular
+      - run: |
+          cd mbed-os-example-cellular
+          git clone https://github.com/ARMmbed/mbed-os.git
+      - run:
+          working_directory: mbed-os-example-cellular
+          command: ln -s $PWD/../mbed-os mbed-os
+      - persist_to_workspace:
+          root: /root/project
+          paths: mbed-os-example-cellular
+
+  build_disco_l496ag:
+    docker:
+      - image: mbedos/mbed-os-env:stable
```

Comment:
try with :latest and see if it has the correct mbed-tools (remove the `ython3 -m pip install mbed-tools --upgrade`)

Reply:
Okay. I'll do it after current CI succeeds/fails.

Reply:
actually the cmake still needs that upgrade even on latest